*** ./smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2013-12-01 10:58:34.284913336 +0800
--- /home/xiangxin/patchrom/miui/v5/st25i/temp/new_smali/framework.jar.out/./smali/com/android/internal/widget/ScrollingTabContainerView$TabClickListener.smali	2013-12-01 11:03:33.598599203 +0800
***************
*** 111,115 ****
--- 111,119 ----
  
      .end local v0           #child:Landroid/view/View;
      :cond_1
+     iget-object v4, p0, Lcom/android/internal/widget/ScrollingTabContainerView$TabClickListener;->this$0:Lcom/android/internal/widget/ScrollingTabContainerView;
+ 
+     invoke-static {v4, p1}, Lcom/android/internal/widget/ScrollingTabContainerView$Injector;->onTabViewClick(Lcom/android/internal/widget/ScrollingTabContainerView;Landroid/view/View;)V
+ 
      return-void
  .end method
