*** ./smali/android/app/INotificationManager.smali	2013-12-01 10:57:45.645911812 +0800
--- /home/xiangxin/patchrom/miui/v5/st25i/temp/new_smali/framework.jar.out/./smali/android/app/INotificationManager.smali	2013-12-01 11:02:38.943758544 +0800
***************
*** 15,20 ****
--- 15,28 ----
  
  
  # virtual methods
+ .method public abstract areNotificationsEnabled(Ljava/lang/String;)Z
+     .annotation system Ldalvik/annotation/Throws;
+         value = {
+             Landroid/os/RemoteException;
+         }
+     .end annotation
+ .end method
+ 
  .method public abstract areNotificationsEnabledForPackage(Ljava/lang/String;)Z
      .annotation system Ldalvik/annotation/Throws;
          value = {
